shared/chat/conversation/messages/text/index.js

```diff
@@ -13,16 +13,26 @@ export type Props = {
   mentionsChannelName: Types.MentionsChannelName,
 }
 
-const MessageText = ({text, type, isEditing, mentionsAt, mentionsChannel, mentionsChannelName}: Props) => (
-  <Markdown
-    style={getStyle(type, isEditing)}
-    meta={{mentionsAt, mentionsChannel, mentionsChannelName}}
-    styleOverride={isMobile ? {paragraph: getStyle(type, isEditing)} : undefined}
-    allowFontScaling={true}
-  >
-    {text}
-  </Markdown>
-)
+const MessageText = ({text, type, isEditing, mentionsAt, mentionsChannel, mentionsChannelName}: Props) => {
+  const markdown = (
+    <Kb.Markdown
+      style={getStyle(type, isEditing)}
+      meta={{mentionsAt, mentionsChannel, mentionsChannelName}}
+      styleOverride={Styles.isMobile ? {paragraph: getStyle(type, isEditing)} : undefined}
+      allowFontScaling={true}
+    >
+      {text}
+    </Kb.Markdown>
+  )
+
+  return Styles.isMobile ? (
+    <Kb.Box2 direction="vertical" style={styles.wrapper}>
```

Comment:
mobile needs this wrapper as markdown itself will export sibling texts inside a fragment so they get laid out based on whatever the parent is. instead we need these to stack vertically